### ISSUE TITLE
pthread_condattr_setclock is not supported on macOS

### ISSUE THREAD

**Issue by aw1621107** [farsightsec/fstrm#34]
Compilation succeeds, but `make check` fails with

```
dyld: lazy symbol binding failed: Symbol not found: _pthread_condattr_setclock
  Referenced from: /private/tmp/fstrm-20170317-1900-13gpt2k/fstrm-0.3.1/fstrm/.libs/libfstrm.0.dylib
  Expected in: flat namespace

dyld: Symbol not found: _pthread_condattr_setclock
  Referenced from: /private/tmp/fstrm-20170317-1900-13gpt2k/fstrm-0.3.1/fstrm/.libs/libfstrm.0.dylib
  Expected in: flat namespace
```

**Comment by cmikk** [comment]
Thank you for the report.

Please check out branch `branches/issue35` and verify that it works for you.

**Comment by aw1621107** [comment]
Yep, all the tests pass! Thanks for the fix!

**Comment by cmikk** [comment]
I've reworked `branches/issue34`, could you verify that it still works for you?

You will need to run `git branch -D branches/isssue34` first, or test it from a fresh checkout.

**Comment by reedjc** [comment]
The fix has been released in the 0.3.2 version. Thanks for reporting this.
